### ISSUE TITLE
Chosen project choice is never taken into account with multiple projects config

### ISSUE THREAD

**Issue by smonff** [Ovid/git-workflow#9]
Hi,

I started using `git lab` a few days ago. I set up a multi-projects configuration with a `.git-lab` file in my home directory.  The config file looks like this

```ini
[gitlab]
project_id=123
url=https://gitlab.smonff.com/api/v4
token=S3kR3t

[preferences]
user=sf
max_length=80

[project Backend]
project_id=456
[project API]
project_id=123
[project Dotfiles]
project_id=789
[project Frontend]
project_id=567
```

As far as I can say, the chosen project that should be set after the menu get displayed is never taken into account: the default project is always used, or an `undef` value is used if no default project exists (in the `[gitlab]` section). 

Looks like it is because the `$project_id` value is never overwritten by the `$chosen` value:

https://github.com/Ovid/git-workflow/blob/105bb1d5176ee29779739cd9a7d9b0039600916b/bin/git-lab#L101

Doing something like this, before returning the `$project_id` should help :

```perl
    my $name = $name_for_id{$chosen};
        say "You chose project $name";
+       $project_id = $chosen;
    }
    return $project_id;
}
```

Note that I didn't try to reproduce the issue with a per-repository `.git-lab` configuration. Maybe the behaviour is different then.

I will follow up with a MR if it makes sense.

**Comment by smonff** [comment]
Closed by #10 .

**Comment by Ovid** [comment]
Thank you again for this, Seb :)
